pkg/vm/vm.go

```diff
@@ -850,17 +855,17 @@ func (v *VM) execute(ctx *Context, op Instruction) {
 		}
 
 	case APPCALL, TAILCALL:
-		if len(v.scripts) == 0 {
-			panic("script table is empty")
+		if v.getScript == nil {
+			panic("no getScript callback is set up")
```

Comment:
dunno if it's planned that we have 2 diff messages:
`panic("no getScript callback is set up")`
and on line 864 869 
`panic("could not find script")`
for the same reason

Reply:
These are different! In the second case you do have the callback, but it can't find the script to call.

Reply:
got it!